admin/services/data-access/group.data.service.js

```diff
@@ -8,10 +8,64 @@ const groupDataService = {}
  * @param query
  * @returns {Promise<Promise|*>}
  */
-groupDataService.fetchGroups = async function (query) {
+groupDataService.getGroups = async function (query) {
   const q = query || {}
   q.isDeleted = false
-  return Group.find(q).sort({ name: 'asc' }).lean().exec()
+  return Group
+    .find(q)
+    .collation({ locale: 'en', strength: 2 })
```

Comment:
this will not work with cosmosDB and will throw an error in the frontend.
it was the reason why I had to block pupils case insensitive sorting.

Reply:
@GeorgeChatzigiannis @pris54 I wonder if we should just park this subtle difference until we tackle it in the data layer refactor?  seems a waste for Paula to spend time on it when it will soon be refactored anyway...

Reply:
I was under the impression that every new PR from this point and forward should have data layer work done in SQL?

Reply:
@GuyHarwood @GeorgeChatzigiannis Thanks for the comments on this issue. My understanding after seeing Guy's first comment was that this issue will be tackled as part of the SQL refactoring. I've added a comment to the PBI explaining this.

Reply:
I don't see how it would have been feasible for Paula to do this in SQL, given the xmas break and the fact that nobody was around to provide guidance on the new services etc.
Now that we are back we should draw a line under CosmosDB.

Reply:
Ok that makes sense

Reply:
@GeorgeChatzigiannis @GuyHarwood This is a branch of the add/edit one ( PR #365 ) which means that this line of code has been commented out with a @TODO comment on top (see PR 365)